### PR TITLE
Increased ping timeout

### PR DESCRIPTION
**Pull request by nvidianz**
### Description

Set ping timeout to 60 (default was 20).

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

**Comment by YuanTingHsieh** [comment]
@nvidianz why do we need this change of adding "None"?

File: nvflare/fuel/f3/drivers/aio_http_driver.py [hunk 1]
```diff
@@ -33,6 +33,7 @@
 log = logging.getLogger(__name__)
 
 THREAD_POOL_SIZE = 8
+PING_TIMEOUT = None
```
[the remark this answers is not in the document]
Comment (YuanTingHsieh) [reply nvflare/fuel/f3/drivers/aio_http_driver.py:36]: @nvidianz ^

[the remark this answers is not in the document]
Comment (nvidianz) [reply nvflare/fuel/f3/drivers/aio_http_driver.py:36]: Nope. Closed.

**Comment by nvidianz** [comment]
Not needed anymore.

**Pull request closed without merging**